### PR TITLE
Correct Helm index keys

### PR DESCRIPTION
**Pull request by sabre1041**
Corrections to how Helm index keys are stored

### CODE REVIEW

File: pkg/types/helm/provenance_test.go [hunk 1]
```diff
@@ -37,12 +38,16 @@ func TestProvenance(t *testing.T) {
...
 	}
 
-	checksum, err := provenance.GetChartHash()
+	algorithm, checksum, err := provenance.GetChartAlgorithmHash()
 
 	if err != nil {
 		t.Fatalf("Error retrieving chart hash: %v", err)
 	}
 
+	if models.AlpineV001SchemaPackageHashAlgorithmSha256 != algorithm {
```
Comment (dlorenc) [inline pkg/types/helm/provenance_test.go]: You should be able to find this in helm somewhere instead of alpine...

**Review by dlorenc — COMMENTED** [review]
Just the one nit about which constant to use.

File: pkg/types/helm/provenance_test.go (hunk 1 above)
Comment (sabre1041) [reply pkg/types/helm/provenance_test.go]: @dlorenc whoops. good catch. updated

File: pkg/types/helm/provenance_test.go [hunk 2]
```diff
@@ -37,12 +38,16 @@ func TestProvenance(t *testing.T) {
...
 
-	checksum, err := provenance.GetChartHash()
+	algorithm, checksum, err := provenance.GetChartAlgorithmHash()
 
 	if err != nil {
 		t.Fatalf("Error retrieving chart hash: %v", err)
 	}
 
+	if models.HelmV001SchemaChartHashAlgorithmSha256 != algorithm {
+		t.Fatalf("Unexpected checksum algorithm. Expected %s, found %s", models.AlpineV001SchemaPackageHashAlgorithmSha256, algorithm)
```
Comment (bobcallaway) [inline pkg/types/helm/provenance_test.go]:
```suggestion
		t.Fatalf("Unexpected checksum algorithm. Expected %s, found %s", models. HelmV001SchemaChartHashAlgorithmSha256, algorithm)
```

Comment (sabre1041) [reply pkg/types/helm/provenance_test.go]: sigh. updated. good old copy/paste similar to the the prior issue

**Review by bobcallaway — APPROVED** [review]

**Pull request merged**